### ISSUE TITLE
 Please complete the previous course before starting the lesson. 

### ISSUE THREAD

**Issue by lvnilesh** [woocommerce/sensei-content-drip#101]
Running this in a vagrant test box.

Created a course with three lessons and no prerequisite courses. 
Added drip schedule

    t = 0 first lesson 
    t + 7 days second lesson
    t+ 14 third lesson

User clicks on first lesson and is alerted with this info message.

    Please complete the previous course before starting the lesson. 

The hyperlink embedded with `previous course` points to `itself.`

Seems like a bug to me.

**Comment by lvnilesh** [comment]
as a crude workaround I can suppress but this is not ideal.

`

/* hide buggy sensei-message info */

.sensei p.sensei-message.info, .sensei div.sensei-message.info, .course-container p.sensei-message.info, .course-container div.sensei-message.info, .course p.sensei-message.info, .course div.sensei-message.info, .lesson p.sensei-message.info, .lesson div.sensei-message.info, .quiz p.sensei-message.info, .quiz div.sensei-message.info, .learner-info p.sensei-message.info, .learner-info div.sensei-message.info 

{

    display: none;
}
`

**Comment by lvnilesh** [comment]
No that crude workaround I thought might work doesn't actually work because it also hides future availability messages.

**Comment by lvnilesh** [comment]
The issue seems to be originating from https://github.com/Automattic/sensei/blob/473abad4baf59bae15724ea84c08dce39035d3a4/includes/class-sensei-lesson.php#L3606

**Comment by lvnilesh** [comment]
I see similar issue in sensei core https://github.com/Automattic/sensei/issues/1307

**Comment by lvnilesh** [comment]
Roll back to sensei 1.9.5 seems to be a workaround.
```
cd sensei
git checkout version/1.9.5 
Previous HEAD position was 4deae19... Merge pull request #1445 from Automattic/release/1.9.6
HEAD is now at 279a2f9... Date bump
```

**Comment by danjjohnson** [comment]
@lvnilesh I can't replicate this following your steps. Are there any other conditions required to see this issue?

**Comment by danjjohnson** [comment]
Also reported in 475695-zd-woothemes

**Comment by lvnilesh** [comment]
It's a core issue. I hacked core like this https://github.com/beacloudgenius/sensei/tree/hack and now it works.

**Comment by lvnilesh** [comment]
and send a hacky pull request https://github.com/Automattic/sensei/pull/1567

**Comment by danjjohnson** [comment]
Closing as the issue is in core and has a fix in place for v1.9.8 https://github.com/Automattic/sensei/issues/1564
